client/src/view/fabric-view.tsx

```diff
@@ -216,13 +219,13 @@ export function FabricView({fabric, selection, setSelection}: {
             })
             const topFace = faces.reverse().pop()
             if (topFace) {
-                setSelection({selectedFace: topFace})
+                setSelection({selectable: Selectable.FACE, selectedFace: topFace})
             }
         }
         return (
             <mesh
                 ref={meshRef}
-                onPointerDown={onPointerDown}
+                onClick={onClick}
```

Comment:
This mightve broken selection, I can't select anything it seems.

Reply:
I can select faces but i have to be careful not to move while clicking.  I was looking for a way to make it more tolerant of a few pixels move.  Define "click".